packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/DatagridWithNoPinnedColumns.scss

```diff
@@ -0,0 +1,7 @@
+@import './Datagrid';
```

Comment:
This is one-to-one styles to `Datagrid.scss`. Lets reuse `Datagrid.scss` and remove this file.